definitions/npm/bitcoinjs-lib_v2.x.x/flow_>=v0.17.x/bitcoinjs-lib_v2.x.x.js

```diff
@@ -97,6 +97,8 @@ declare module 'ecurve' {
 }
 // ---------- copypasta end ---- 
 
+import {Point as ECPoint} from 'ecurve';
```

Comment:
You cannot use imports within lib defs like this, which is why there is TODO at the top of this file.